x-pack/legacy/plugins/apm/cypress/cypress/integration/apm.feature

```diff
@@ -0,0 +1,9 @@
+Feature: APM
+
+  Background:
+    Given a user browses the APM UI application
+    When the user inspects the opbeans-go service
+
+  Scenario: Transaction duration charts
+    Then should redirect to correct path with correct params
+    And should have correct y-axis ticks
```

Comment:
The `And` clause could be used after a `Then` or a `When`. It's indistinguishable from any of them in the sense, but it's important the semantics of And steps: it's like creating another assertion, which ideally must be done just once.

Reply:
And works with Given, When or Then it makes a scenario easier to read.

Given a thing
And another thing

Reads better than

Given a thing
Given another thing